compressors/integer_coders.py

```diff
@@ -85,3 +86,149 @@ def test_universal_uint_encode_decode():
 
     is_lossless, _ = try_lossless_compression(data_block, encoder, decoder)
     assert is_lossless
+
+
+"""
+shared initialization for Golomb encoder and decoder
+"""
+
+
+def _initialize_golomb_coder(coder, M: int):
+    assert M > 0
+    coder.M = M
+    coder.b = int(math.floor(math.log2(coder.M)))
```

Comment:
I believe the names `M` and `b` are from wikipedia article.. but perhaps better to use more descriptive names or add a comment to what they mean

Reply:
Not sure they have a name. I will add a comment.

Reply:
sounds good